app/views/games/_index_dev.html.erb

```diff
@@ -4,19 +4,22 @@
 <div class="wrapper">
     <div class="one"><%= display_picture g %></div>
     <div class="two">
-        <h3 style="font-weight: bold; color: #EF7030;"><%= link_to g.title, game_path(g), class: "game-title" %></h3>
+        <h3><%= link_to g.title, game_path(g), class: "game-title" %></h3>
```

Comment:
Variable `g` is too short and ambiguous. I'm guessing it's supposed to represent a `game`. Could rename occurrences of `g` as such? 😃